### ISSUE TITLE
QOS

### ISSUE THREAD

**Issue by Garthmj** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#779]
What exactly are the QOS setting that should be on the FW/Router? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c400a556-211c-1b7e-73b9-701c842d9671
* Version Independent ID: 7ad16fa2-0c99-52dd-6c0b-66da0077ff16
* Content: [ExpressRoute and QoS in Skype for Business Online](https://docs.microsoft.com/en-us/skypeforbusiness/optimizing-your-network/expressroute-and-qos-in-skype-for-business-online)
* Content Source: [Skype/SfbOnline/optimizing-your-network/expressroute-and-qos-in-skype-for-business-online.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbOnline/optimizing-your-network/expressroute-and-qos-in-skype-for-business-online.md)
* Service: **skype-for-business-online**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by AndreaBarr** [comment]
@Garthmj Thank you for this feed back

I agree that the QOS settings should be in the article. 

I was able to get this into the backlog for the Teams/Skype for Business team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.

**Comment by Garthmj** [comment]
I'm curious why close this item, when it is not completed? It should be left open so that the task is not lost. IMO closing this means to me, that is is not a priority and it will never get added to the docs. 

I looking for these details now as Skype is ALWAYS having problems connected to MS Skype calls, in many cases Skype (audio) is unusable where as GTM almost never seems to have these problem.

**Comment by AndreaBarr** [comment]
@Garthmj Our process is to put it in the backlog where we track our work and close the GitHub ticket. Once the backlog is finished or there is an update we comment here to let you know. This way the open GitHub issues are new issues or issues we thought were resolved but are not and need extra attention.

**Comment by get-itips** [comment]
Hi @Garthmj hope you are doing great! thanks for your feedback and comments, we really appreciate them.
QoS Settings for router are present in the Recommended DSCP settings table, are you referring to these settings? If so, please tell me if this information has helped you.

Many thanks

**Comment by Garthmj** [comment]
Can you share the direct link to the Docs page?

**Comment by get-itips** [comment]
Hi @Garthmj you will find it under https://docs.microsoft.com/en-us/skypeforbusiness/optimizing-your-network/expressroute-and-qos-in-skype-for-business-online#part-2-expressroute-skype-for-business-qos hope that helps you

**Comment by yogkumgit** [comment]
@get-itips  Thank you very much for the contribution and sharing this explanation. @Garthmj 
Hope this comment is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
